File: 3753-maximum-difference-between-even-and-odd-frequency-i/3753-maximum-difference-between-even-and-odd-frequency-i.py
class Solution:
    def maxDifference(self, s: str) -> int:
        arr = [0]*26
        odd, even = 0, 1000
        for c in s:
            arr[ord(c) - ord('a')] += 1
        
        for i in arr:
            if(i % 2 == 0 and i != 0):
                even = min(even, i)
            else:
                odd = max(odd, i)
                
        return odd - even

File: 3753-maximum-difference-between-even-and-odd-frequency-i/3753-maximum-difference-between-even-and-odd-frequency-i.java
class Solution {
    public int maxDifference(String s) {
        int arr[] = new int[26]; 
        int even = 1000, odd = 0;
        for(char i : s.toCharArray()){
            arr[i - 'a']++;
        }
        for(int i : arr){
            if(i%2 == 0 && i != 0)
                even = Math.min(even, i);
            else    
                odd = Math.max(odd, i); 
        }
        return odd - even;
    }
}